### PR TITLE
Fix social user info saves

### PR DESCRIPTION
**Pull request by gmethvin**
Fix a few save calls I noticed to use the slickified version

@andrewconner can tell me if I'm on the right track here and point out anything else I should look for

### CODE REVIEW

File: shoebox/test/com/keepit/common/social/SocialUserCreateConnectionsTest.scala [hunk 1]
```diff
@@ -44,11 +34,16 @@ class SocialUserCreateConnectionsTest extends SpecificationWithJUnit {
...
 
-        val socialUserInfo = CX.withConnection { implicit conn =>
-          SocialUserInfo(fullName = "Bob Smith", socialId = SocialId("bsmith"), networkType = SocialNetworks.FACEBOOK).withUser(User(firstName = "fn1", lastName = "ln1").save).save
+        val socialUserInfo = inject[DBConnection].readWrite { implicit c =>
+          inject[SocialUserInfoRepo].save(SocialUserInfo(
+            fullName = "Bob Smith",
+            socialId = SocialId("bsmith"),
+            networkType = SocialNetworks.FACEBOOK
+          ).withUser(
+            CX.withConnection { implicit c => User(firstName = "fn1", lastName = "ln1").save }
```
Comment (andrewconner) [inline shoebox/test/com/keepit/common/social/SocialUserCreateConnectionsTest.scala]: You should use Slick here as well. Anywhere with `CX.withConnection` is Circumflex.

File: shoebox/test/com/keepit/common/social/SocialUserCreateConnectionsTest.scala [hunk 2]
```diff
@@ -62,8 +57,14 @@ class SocialUserCreateConnectionsTest extends SpecificationWithJUnit {
...
```
Comment (andrewconner) [inline shoebox/test/com/keepit/common/social/SocialUserCreateConnectionsTest.scala]: Same here.

File: shoebox/test/com/keepit/common/social/SocialUserCreateConnectionsTest.scala (hunk 1 above)
Comment (gmethvin) [inline shoebox/test/com/keepit/common/social/SocialUserCreateConnectionsTest.scala]: fixed

Comment (andrewconner) [inline shoebox/test/com/keepit/common/social/SocialUserCreateConnectionsTest.scala]: Whoops, breaks tests.

**Pull request merged**